src/Template/Sadrs/xml/e2b.ctp

```diff
@@ -134,7 +141,7 @@
             <receiveremailaddress/>
         </receiver>
         <patient>
-            <patientinitial><?php echo $sadr['patient_name']; ?></patientinitial>
+            <patientinitial><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '', $sadr['patient_name']); ?></patientinitial>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate